Source/Meadow.Foundation.Peripherals/ICs.IOExpanders.Pcx857x/Samples/Pca8574_Sample/meadow.config.yaml

```diff
@@ -0,0 +1,2 @@
+﻿MonoControl:
```

Comment:
I think we can remove these to avoid confusion. JIT is enabled by default now.

Reply:
we should bulk remove